catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -138,6 +145,20 @@ public void setQuerySources(Set<String> querySources) {
     }
   }
 
+  public void setAttributeOverrides(Set<String> attributeOverrides) {
+    this.attributeOverrides.clear();
```

Comment:
@troymohl why do we iteratively clear and add to these sets?  do the lists have to be final?

Reply:
I did it so a null list wasn't possible.